nephio-ansible-install/roles/gitea/create/tasks/main.yaml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Get UID
+  getent:
+    database: passwd
+  when:
+    - gitea_username is defined
+    - gitea_password is defined
+- name: Create a gitea directory
+  ansible.builtin.file:
+    path: /home/{{ cloud_user}}/gitea
+    state: directory
+    mode: '0755'
+- name: Start gitea container
+  docker_container:
```

Comment:
seems like the module docker_container is not getting recognized when I run the playbook on Ubuntu VM

```
vjayaram@wim-pr-test3:~/nephio-ansible-install$ ansible-playbook playbooks/create-gitea.yaml --connection=local 
ERROR! couldn't resolve module/action 'community.general.docker_container'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/vjayaram/nephio-ansible-install/roles/gitea/create/tasks/main.yaml': line 15, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Start gitea container
  ^ here
```

Reply:
I was able to resolve the issue when I ran `ansible-galaxy collection install community.docker`, so the README needs an update. I have left a comment in the README as well

Reply:
fixed https://github.com/nephio-project/one-summit-22-workshop/pull/106/commits/92affdf9ed2c2a920e7843266952f4bd89edc5f3